GitUI/CommandsDialogs/FormSparseWorkingCopyViewModel.cs

```diff
@@ -231,7 +231,7 @@ private void SaveChangesTurningOffSparseSpecialCase()
             }
 
             // Adjust the rules
-            // Comment out all existing nonempty lines, add the single �/*� line to make a total pass filter
+            // Comment out all existing nonempty lines, add the single �/*� line to make a total pass filter
```

Comment:
⚠️ An encoding error occurred, but in this case I recommend changing the comment to use `"` instead of curly-quotes.